### PR TITLE
Add monthly update for February 2024

### PR DESCRIPTION
**Pull request by majetideepak**
Resolves: https://github.com/facebookincubator/velox/issues/9299

### CODE REVIEW

File: velox/docs/monthly-updates/february-2024.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* Add stats for null skew in join operator.
+* Add background profiler that starts Linux perf on the Velox process.
+* Fix ``out of range in dynamic array`` error in Task::toJson.
+* Delete unused ``max_arbitrary_buffer_size`` config.
+
+Presto Functions
+================
+
+* Add :func:`typeof`, :func:`from_iso8601_date` scalar functions.
+* Add support for deserializing a single column in Presto page format.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:18]: This item doesn't belong to "Presto Functions" section. Let's move it to "Core Library".

File: velox/docs/monthly-updates/february-2024.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* Add support for deserializing an all-null column serialized as UNKNOWN type in Presto page format.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:19]: ditto

File: velox/docs/monthly-updates/february-2024.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* Add support for DECIMAL input type to :func:`set_agg` and :func:`set_union`.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:20]: add "aggregate functions." to the end of the sentence

File: velox/docs/monthly-updates/february-2024.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* Add support for UNKNOWN input type to :func:`checksum`.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:21]: ditto

File: velox/docs/monthly-updates/february-2024.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+* Add support for :func:`date` +/- INTERVAL YEAR MONTH.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:22]: typos? What does it mean to add / substract from a function?

File: velox/docs/monthly-updates/february-2024.rst [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+* Improve :func:`parse_datetime` to allow ``UCT|UCT|GMT|GMT0`` as ``Z``
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:23]: Improve -> Add support for UCT|....

File: velox/docs/monthly-updates/february-2024.rst [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+* Convert TIMESTAMP_WITH_TIME_ZONE type to a primitive type.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:24]: Doesn't belong to functions section. Let's move to Core Library.

File: velox/docs/monthly-updates/february-2024.rst [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+===============
+
+* Add :spark:func:`array_repeat`, :spark:func:`date_from_unix_date`, :spark:func:`weekday`, :spark:func:`minute`, :spark:func:`second` scalar functions.
+* Add :spark:func:`ntile` window function.
+
+Hive Connector
+==============
+
+* Add ``ignore_missing_files`` config.
+* Add write support in ABFSFileSystem.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:36]: Add write support to ABFS file system.

File: velox/docs/monthly-updates/february-2024.rst [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+* Add support for proxy in S3FileSystem.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:37]: Add support for proxy to S3 file system.

File: velox/docs/monthly-updates/february-2024.rst [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+=====
+
+* Add support to export UNKNOWN type to Arrow array.
+* Add support to convert Arrow REE arrays to Velox Vectors.
+
+Performance and Correctness
+===========================
+
+* Add FieldReference benchmark.
+* Add fuzzer test for Window functions.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:49]: Add Window fuzzer. 

Perhaps, add a link to documentation.

File: velox/docs/monthly-updates/february-2024.rst [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+* Fix ``Too many open files`` error in JoinFuzzer.
```
Comment (mbasmanova) [inline velox/docs/monthly-updates/february-2024.rst:50]: JoinFuzzer -> Join fuzzer

**Review by mbasmanova — COMMENTED** [review]
@majetideepak Deepak, thank you for putting together this update. Looks great % a couple nits.

**Comment by majetideepak** [comment]
@mbasmanova thanks for the feedback. Can you take another look?

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Pull request closed without merging**